README.md

```diff
@@ -1,9 +1,113 @@
 # Simple data analysis
 
-This project has two main goals:
+These project's goals are:
 
-- To encourage more data journalists, academics and web developers to dive into the wonderful world of data analysis and visualization in JavaScript.
+- To ease the way for non-coders (especially journalists) into the beautiful world of data analysis and data visualization in JavaScript.
 
-- To accelerate my workflow (and hopefully yours) with a set of easy to use functions based on the best libraries out there.
+- To uniformize and accelerate frontend/backend workflows with a simple to use library working both in the browser and with NodeJS.
 
-The project is under development. Stay tuned.
\ No newline at end of file
+We are always trying to improve it. Feel free to start a conversation or open an issue. Pull requests are welcome as well!
+
+# Table of contents
+
+1. [Core principles](#core-principles)
+2. [Easiest way to use](#the-easiest-way-to-use-the-library)
+3. [Importing from HTML](#importing-from-the-html)
+4. [NodeJS and JavaScript bundlers](#working-with-nodejs--javascript-bundlers)
+5. [SimpleDocument (experimental)](#simpledocument-experimental-for-nodejs-only)
+6. [All functions and methods](#all-functions-and-methods-available)
+
+## Core principles
+
+The library expects **tabular data** stored in CSV files or **arrays of objects** stored in JSON files. It works best when the data is tidy:
+
+1. Every column (or key) is a variable
+
+2. Every row (or item) is an observation
+
+3. Every cell (or value) is a single value
```

Comment:
Perhaps we should give an example? 

```suggestion

1. Every column (or key) is a variable 

2. Every row (or item) is an observation

3. Every cell (or value) is a single value

For instance, for hourly meteorological data, our tidy dataset would look like this.  
time         day            temperature   precipitation
00:00:00     2022-05-02      15.2          0
01:00:00     2022-05-02      15.9.         0
02:00:00     2022-05-02      16.2.         0
03:00:00     2022-05-02      18.0          0.5
04:00:00     2022-05-02      19.3.         0
05:00:00     2022-05-02      19.4.         0
```

Reply:
I think I'll link to a resource explaining it. Otherwise, it could be long before we actually get to the code!